.env.example

```
@@ -8,17 +8,10 @@ CLIENT_SECRET=MCP_password_here
 # Slack configuration
 SLACK_BOT_TOKEN=your_slack_bot_token_here
 SLACK_SIGNING_SECRET=your_slack_signing_secret_here
-SLACK_WEBHOOK_URL=your_slack_webhook_url_here
-
-# Port for the application to listen on (default: 3001)
-PORT=3001
+SLACK_APP_TOKEN=your_slack_app_token_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SLACK_APP_TOKEN key should go before the SLACK_BOT_TOKEN key

(UnorderedKey)